backend/cms/views/pages/page_form.py

```diff
@@ -222,9 +218,14 @@ def save(self, *args, **kwargs):
             page_translation.creator = user
             page_translation.language = self.language
 
+        page_translation.version = page_translation.version + 1
+        page_translation.id = None
```

Comment:
Instead of setting the `id = None`, we can alternatively call `save(force_insert=True)`

Reply:
Yes, using `save(force_insert=True)` sounds good.